flagsmith/polling_manager.py

```diff
@@ -14,12 +16,12 @@
 
 
 class EnvironmentDataPollingManager(threading.Thread):
-    def __init__(
+    def __init__(  # type: ignore
```

Comment:
Please clarify the error you're ignoring here.

Reply:
I was ignoring `*args` and `**kwargs`, added typing for it.